sources/packages/web/src/types/vuetify.ts

```diff
@@ -36,3 +36,14 @@ export type VForm = Vue & {
    */
   errors: ErrorMessage[];
 };
+
+/**
+ * Vuetify file for a v-file-input component.
+ */
+export interface FileInputFile extends Blob {
```

Comment:
IMO the interface can be name as InputFile.

Reply:
Agree, it sounds better.

Reply:
Renamed.